app/models/user.rb

```diff
@@ -416,6 +416,14 @@ def add_subscriptions_token
     update!(subscriptions_token: SecureRandom.base58(32)) if subscriptions_token.blank?
   end
 
+  def self.maximum_attempts
+    (Tenant.current_secrets.security[:lockable][:maximum_attempts] || 20).to_i
```

Comment:
This version is giving me problems with other specs. Why not the [Cabildo Teneride implementation](https://git.nubarron.teide.int/shs-consul/cabildoabierto-consul/-/merge_requests/28/diffs#0c788cab1662321bd1e65ad8c84fbe4b6008c822_17_16)?

```
  1) Users::SessionsController access logs when feature is enabled when a sign in process fails it calls the authentication logger
     Failure/Error: (Tenant.current_secrets.security[:lockable][:maximum_attempts] || 20).to_i
     
     NoMethodError:
       undefined method `[]' for nil:NilClass
     
           (Tenant.current_secrets.security[:lockable][:maximum_attempts] || 20).to_i
                                           ^^^^^^^^^^^
     # ./app/models/user.rb:420:in `maximum_attempts'
```

Reply:
Great that you spotted this!

In the Cabildo a change was made that does not have this PR. I have just added it.

The difference is to use "dig" so that in case of not having the secret defined it does not fail.

```Tenant.current_secrets.dig(:security, :lockable, :maximum_attempts)```